README.md

```diff
@@ -89,40 +82,84 @@ df
+
+#### 2. Running some algorithms 
+
+| Type | Location  | Description |
+| ------------- | ------------- | ------------- |
+| Example | <a href="https://docs.raphtory.com/en/master/_autodoc/com/raphtory/algorithms/generic/centrality/index.html">centrality</a> | Centrality algorithms for finding important nodes |
+| Example | <a href="">community</a> | Community detection for finding clusters |
+| Example | <a href="https://docs.raphtory.com/en/master/_autodoc/com/raphtory/algorithms/generic/dynamic/index.html">dynamic</a> | Dynamic algorithms and random walks |
+| Example | <a href="">temporal</a> | Applying time magic to find historic and future trends |
+
+#### 3. Developing an end-to-end application
+
+| Type | Location  | Description |
+| ------------- | ------------- | ------------- |
+| Notebook | <a href="https://github.com/Raphtory/Raphtory/blob/master/examples/nft/src/main/python/nft_analysis.ipynb">nft_analysis.ipynb</a> | Analysing pump and dump cycles of popular NFTs |
+| Notebook | <a href="https://github.com/Raphtory/Raphtory/blob/master/examples/companies-house/src/main/python/PPEContractsAnalysisNotebook.ipynb">ppe_analysis.ipnyb</a>  | Fraud and COVID-19 Relief Schemes |
 
 # Want to do something more complex?
-If you would like to do something more complex, follow these links:
+Learn more about Raphtory and how to use it to build your analysis and project by following these links.
 
-- **[Documentation](https://docs.raphtory.com/)**
-- [Complete list of available algorithms in Raphtory](https://docs.raphtory.com/en/master/_autodoc/com/raphtory/algorithms/generic/index.html)
+- **[Latest documentation](https://docs.raphtory.com/)**
+- [Using Raphtory in 100 seconds](https://docs.raphtory.com/en/master/Introduction/ingestion.html)
+- [Complete list of available algorithms](https://docs.raphtory.com/en/master/_autodoc/com/raphtory/algorithms/generic/index.html)
 - [Writing your own algorithm in Raphtory](https://docs.raphtory.com/en/master/Analysis/LOTR_six_degrees.html)
 
 # Bounty board
 
-We are currently running a bounty board for anyone that wishes to contribute to Raphtory, whether it be cool features or yet to be implemented algorithms.
+Raphtory is currently offering rewards for contributions, such as new features or algorithms. Contributors will receive swag and prizes! 
 
-All contributors receive swag and small prizes!
-
-To get you started, we currently have a list of algorithms that we would like to build into Raphtory. With a number of 🍇 low hanging fruit ones. Make sure to check it out at https://www.raphtory.com/algorithm-bounty/ 
+To get started, check out our list of desired algorithms at https://www.raphtory.com/algorithm-bounty/ which include some 🍇 low hanging fruit that are easy to implement. 
 
 
 # Community  
+Join the growing community of open-source enthusiasts using Raphtory to power their graph analysis projects!
+
+- Follow 
```

Comment:
this should go on one line with the text, looks funny otherwise and wastes alot of space